### ISSUE TITLE
BUG: fix the view current view selection on layout with N views for the microscopy extension

### ISSUE THREAD

**Issue by Punzo** [OHIF/Viewers#2156]
### Describe the Bug

For the miscroscopy extension, when having a layout with N (with N> 1) views, it is not possibile to change the active view by clicking on the desidered view 

### What steps can we follow to reproduce the bug?

1. open a miscropy datasets with more than one channel
2. set a layout with more than one view
3. click on a view to activate it
4. the view does not get activated when clicking on the view.

### Expected behavior

the selected view gets activated when clicking on it.

**Comment by Punzo** [comment]
@pieper since we will use the slim viewer, I guess this is not anymore a HTAN/IDC priority, isn't it? 

I will leave this open, but I will remove the HTAN IDC labels

**Comment by pieper** [comment]
Right, this particular bug is no longer important and can probably be closed unless we think it's a problem for other OHIF use cases.

But we will want to have a layout feature in Slim, so we should open up an issue there.  Actually, now that I think about it we should develop a general list of functions we expect from Slim and use that to prioritize development.  Perhaps we could develop it as a generalization of [your meeting notes](https://docs.google.com/document/d/1LM457qPBG1JIlwCDQQOQ1XbRC0Rz9Ea5QmN9aHny1lw/edit).

**Comment by Punzo** [comment]
Ok, thanks for the feedback. I would leave open since the bug is there, I would say it's just that it is not anymore a HTAN/IDC priority. 

Yes definitely at next IDC-OIHF meeting we should talk about the slim viewer UI (layouts, multi-channels/blending controls, etc...), design what we need and add a section in the Google document about the interface / create issues in the slim repository. Pinging also @fedorov just for reference.

**Comment by sedghi** [comment]
@igoroctaviano What is the status for this?

**Comment by igoroctaviano** [comment]
@sedghi now IDC uses SLIM for microscopy. @fedorov can we close this legacy issue?

**Comment by sedghi** [comment]
@fedorov @igoroctaviano bump

**Comment by igoroctaviano** [comment]
@sedghi IDC is using SLIM viewer for pathology. Feel free to close this one.

**Comment by fedorov** [comment]
Sorry I missed this earlier. Thank you for closing it!
